doc/nrf/known_issues.rst

```diff
@@ -1853,20 +1853,20 @@ Secure Partition Manager (SPM)
 ==============================
 
 .. note::
-    As of the |NCS| v2.1.0, Secure Partition Manager (SPM) is deprecated and replaced by :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
+    The Secure Partition Manager (SPM) was deprecated in |NCS| 2.1.0 and removed in |NCS| v2.3.0, it is replaced by :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
```

Comment:
Sorry, I missed this one in the first round. It's better to split it in two sentences.
```suggestion
    The Secure Partition Manager (SPM) was deprecated in |NCS| 2.1.0 and removed in |NCS| v2.3.0. It is replaced by :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
```